molecule/default/tests/test_create_and_assign_floating_ip.py

```diff
@@ -56,21 +39,35 @@ def test_create_floating_ip(host):
 def test_assign_floating_ip_to_instance(host):
     """Assign floating IP to an instance/server"""
 
-    create_instance_from_image(host)
+    # Creating an instance from image
+    data = {
+        "instance_name": instance_name,
+        "from_source": 'image',
+        "source_name": image_name,
+        "flavor": flavor,
+        "network_name": private_net,
+    }
+
+    helpers.create_instance(data, host)
+    sleep(30)  # TODO: will find out a better way to avoid implicit sleep.
```

Comment:
Is this sleep statement necessary given the retry nature of the `get_expected_value` calls below?